website/src/views/layout/GlobalSearchContainer.test.tsx

```diff
@@ -61,29 +65,22 @@ describe('GlobalSearchContainer', () => {
   beforeEach(() => {
     mockDom();
 
-    // Replace fetchVenueList with a noop action to stop it from firing API requests
+    // Replace fetchVenueList and useMediaQuery with a noop action to stop it from firing API requests
     mockedFetchVenueList.mockImplementation(() => initAction() as never);
+    mockedUseMediaQuery.mockImplementation(() => initAction() as never);
   });
 
   afterEach(() => {
     mockedFetchVenueList.mockReset();
+    mockedUseMediaQuery.mockReset();
     mockDomReset();
   });
 
   test('hides module when screen size is small', () => {
-    let onMediaChangeCallback: () => void;
-    const addEventListener = (_type: string, listener: (...args: unknown[]) => void) => {
-      onMediaChangeCallback = listener;
-    };
-
-    mockWindowMatchMedia({ matches: true, addEventListener });
-    const { container } = make();
-    expect(container).not.toBeEmptyDOMElement();
-
-    mockWindowMatchMedia({ matches: false, addEventListener });
-    // Trigger render when matches changes
-    act(() => onMediaChangeCallback());
-    expect(container).toBeEmptyDOMElement();
+    expect(mockedUseMediaQuery).not.toHaveBeenCalled();
+    mockWindowMatchMedia({ matches: true });
+    make();
+    expect(mockedUseMediaQuery).toHaveBeenCalled();
```

Comment:
Hmm I don't think we should mock the hook, because:

1. It's an implementation detail of GlobalSearchContainer.
1. We can easily set up the DOM and let the hook do its job.

This should be enough for a sanity check. We test the `matches: true` case in our other test cases anyway.

```suggestion
    mockWindowMatchMedia({ matches: false });
    const { container } = make();
    expect(container).toBeEmptyDOMElement();
```